src/main/java/FigureSupplier.java

```diff
@@ -0,0 +1,37 @@
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int AMOUNT_OF_FIGURES = 5;
+    public static final int BORDER = 5;
+    private Random random = new Random();
+
+    public Figure getRandomFigure() {
+        int randomNumber = random.nextInt(AMOUNT_OF_FIGURES - 1);
```

Comment:
if you're decreasing it by 1, why don't you just change AMOUNT_OF_FIGURES value from 5 to 4?

Reply:
It's a border between 1 (including) and 5 (not including)